src/praisonai/tests/unit/ui/test_ui_bind_aware_creds.py

```
@@ -0,0 +1,165 @@
+    
+    @patch.dict(os.environ, {
+        "CHAINLIT_USERNAME": "admin",
+        "CHAINLIT_PASSWORD": "admin"
+    })
+    def test_accidental_production_deploy_blocked(self):
+        """Test accidental production deploy with default creds is blocked."""
+        mock_app = MagicMock()
+        
+        # Should block (prevents shipping with admin/admin to LAN/internet)
+        with pytest.raises(UIStartupError):
+            register_password_auth(mock_app, bind_host="0.0.0.0")
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)